docs/guides/cli/01_installation.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: installation
+title: Installation
+---
+
+## Overview
```

Comment:
Avoid the first headline as it conflicts with the H1 headline :)

<img width="984" alt="Screenshot 2022-03-28 at 17 13 07" src="https://user-images.githubusercontent.com/3372410/160430009-a92893ec-711f-4710-90c5-30111e1a696a.png">

Reply:
Yea, you're right. I typically wouldn't even "dare" to do that but with our layout, I feel that the initial paragraph might get lost w/o its own heading. But yea, I'll remove it.